unit_tests/wind_energy/test_abl_src.cpp

```diff
@@ -199,7 +199,8 @@ TEST_F(ABLMeshTest, geostrophic_forcing)
         geostrophic_forcing(lev, mfi, bx, amr_wind::FieldState::New, src_arr);
     });
 
-    constexpr amrex::Real corfac = 2.0 * amr_wind::utils::two_pi() / 86400.0;
+    constexpr amrex::Real corfac =
+        4.0 * amr_wind::utils::two_pi() / 86164.091 * 0.80901699437;
```

Comment:
the factor of 4, instead of 2, would need to be reverted back when the m_omega calculation is fixed

Reply:
On it